### ISSUE TITLE
Pagination not working with php-mysql-json-mustache-demo.html?

### ISSUE THREAD

**Issue by mrgusmuller** [no81no/jplist#81]
Hello!,

I'm copied the source code from https://jplist.com/datasourcesexamples/php-mysql-json-mustache-demo  and the json file used on that page.. and created local files for this test to test with MAMP...
On JSON file I'v copied&paste the data entries and updated the 'count' number'.

Pagination doesn't work. I got all them displayed on the same page.. :(

Tnks

**Comment by mrgusmuller** [comment]
OK!,

I'v realised that the JSON example doesn't open an existing JSON file with all the data, but uses a PHP to generate on every request (page change).

It would be nice to have a solution where jplist + mustache could handle with a huge JSON file with all the metadata/information on it! :)

tnks

**Comment by no81no** [comment]
Hi,
In this case you can convert this JSON to HTML only once BEFORE jplist call.
I believe it should not be part of jPList API.
Best regards,
